ethereum/deploy.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "=========== Compiling ==========="
+
+echo "Replacing the deploy commit hash..."
+# Set the deploy commit hash in the contract (used for debugging purposes)
+sed -i "s/__DEPLOY_COMMIT_HASH_PLACEHOLER__/$(git rev-parse HEAD)/g" ./contracts/pyth/Pyth.sol
+
+echo "Building the contract..."
+# Ensure that we deploy a fresh build with up-to-date dependencies.
+rm -rf build && npx truffle compile --all
+
+echo "Adding network metadata to the contract"
+# Merge the network addresses into the artifacts, if some contracts are already deployed.
+npx apply-registry
+
+while [[ $# -ne 0 ]]; do
+    NETWORK=$1
+    shift
+
+    echo "=========== Deploying to ${NETWORK} ==========="
```

Comment:
Could we add a docstring at the top of this file explaining the possible values of `$NETWORK`, and an example invocation of this script?